### PR TITLE
Stop the most obvious comment spam by introducing length limits

### PR DESCRIPTION
**Pull request by baltpeter**
This was bound to happen at some point. *sigh* Though I am actually surprised at how long we got away with having no spam protection at all, so that's something least.

### CODE REVIEW

**Comment by baltpeter** [comment]
Proper handling for this error is implemented in https://github.com/datenanfragen/website/pull/1116, which should be merged first.

File: src/handlers/comments/putComment.js [hunk 1]
```diff
@@ -3,7 +3,13 @@ const nodemailer = require('nodemailer');
 const config = require('../../../config.json');
 const { stripTags } = require('../../util/functions');
 
+const successMessage =
+    'Successfully added comment. It will need to be accepted by an administrator before it is published.';
+
 async function putComment(request, h) {
+    // Silently drop ridiculously short comments, which are pretty much guaranteed to be spam.
+    if (request.payload.message?.length < 5) return { message: successMessage };
```
Comment (mal-tee) [inline src/handlers/comments/putComment.js:11]: This create a timing-based side-channel, but we don't really care. :D

**Review by mal-tee — APPROVED** [review]
Thank you!

File: src/handlers/comments/putComment.js (hunk 1 above)
Comment (baltpeter) [reply src/handlers/comments/putComment.js:11]: If we didn't want people to know that we drop very short comments, I feel like publishing the source would be the bigger problem. :P But, indeed we don't.

**Pull request merged**